monitor/pubsubmon/pubsubmon.go

```diff
@@ -124,7 +127,7 @@ func (mon *Monitor) logFromPubsub() {
 				logger.Error(err)
 				continue
 			}
-			logger.Debugf(
+			logger.Infof(
```

Comment:
Why?
This overwhelms console logs, making it difficult to see useful information

Reply:
ops this is a leftover